python/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenTracing support for gRPC in Python
+
+The `grpcio-opentracing` package makes it easy to add OpenTracing support to 
+gRPC-based systems in Python.
+
+## Installation
+
+```
+pip install grpcio-opentracing
+```
+
+## Getting started
+
+See the below code for basic usage or [examples/trivial](examples/trivial) for a
+complete example.
+
+### Client-side usage example
+
+```python
+tracer = # some OpenTracing Tracer instance
+interceptor = open_tracing_client_interceptor(tracer)
```

Comment:
This was to be consistent with the rest of gRPC where the `__init__.py` file only exposes interface classes and uses functions to construct concrete classes. (See for example the [server function](https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/__init__.py#L1319) that does the same thing.)

Not entirely sure, but I would guess the reasons for this are to keep the documentation in a single location and to have a convention that allows easily supporting additional constructor functions if needed.